### PR TITLE
Feat: habits page

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs

- Habits page added
- Add/ Edit/ Delete a habit
- Categories UI added
- Add Habit model added

## Deploy Preview
https://deploy-preview-3--ripple-habits.netlify.app/

### CODE REVIEW

**Review by manojncsoni007 — APPROVED** [review]
Everything looks good to me.
- also check the reason for failed deployed before merging

File: src/components/Habits/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			<ul className="rui-drawer-content--list no-list p-5">
+				<li className="rui-drawer-content">
+					<label
+						className={`flex-row align-center flex-wrap flex-gap-1 cursor-pointer rui-drawer-content--text h3 link-active`}
+					>
+						<input
+							className="filters rui-drawer-content--text"
+							type="checkbox"
+							value="All"
+							// checked={productsState.categoryFilters[name] ? true : false
```
Comment (anshtripathi01) [inline src/components/Habits/Category.jsx:23]: You commented this. If it's not required remove it.

File: src/components/Habits/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+				</li>
+				<li className="rui-drawer-content">
+					<label
+						className={`flex-row align-center flex-wrap flex-gap-1 cursor-pointer rui-drawer-content--text h3`}
+					>
+						<input
+							className="filters rui-drawer-content--text"
+							type="checkbox"
+							value="All"
+							// checked={productsState.categoryFilters[name] ? true : false
```
Comment (anshtripathi01) [inline src/components/Habits/Category.jsx:40]: Here also

**Review by anshtripathi01 — APPROVED** [review]
Code structure and UI looks good.Go ahead

**Pull request merged**